<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1591</width>
    <height>938</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="mainPageLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="QTabWidget" name="viewTabWidget">
        <property name="minimumSize">
         <size>
          <width>1500</width>
          <height>0</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="imageViewTab">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <attribute name="title">
          <string>Image View</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QTabWidget" name="imageTypeSelector">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="rawImage">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <attribute name="title">
                <string>Image</string>
               </attribute>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout">
                  <item>
                   <widget class="GraphicsView" name="rawImageView"/>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                     <widget class="QSlider" name="rawDisplayScale">
                      <property name="minimum">
                       <number>1</number>
                      </property>
                      <property name="maximum">
                       <number>255</number>
                      </property>
                      <property name="value">
                       <number>255</number>
                      </property>
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_2">
                      <property name="text">
                       <string>Display Scale </string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSpinBox" name="rawDisplayScaleSpinBox">
                      <property name="maximum">
                       <number>255</number>
                      </property>
                      <property name="value">
                       <number>255</number>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QGroupBox" name="rawImageSettings">
                  <property name="minimumSize">
                   <size>
                    <width>240</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="title">
                   <string>Settings and Control</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="frameSliderLayout">
              <item>
               <widget class="QScrollBar" name="frameSelector">
                <property name="minimumSize">
                 <size>
                  <width>700</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="pageStep">
                 <number>1</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="invertedAppearance">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="frameNumberSpinBox">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="frameIndexLabel">
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Frame Index</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="playbackRate">
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>30</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="fpsLabel">
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>FPS</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <layout class="QHBoxLayout" name="controlButtonLayout">
                <item>
                 <layout class="QHBoxLayout" name="frameNavLayout">
                  <item>
                   <widget class="QPushButton" name="playButton">
                    <property name="font">
                     <font>
                      <family>Arial</family>
                      <pointsize>12</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>PLAY</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="prevButton">
                    <property name="font">
                     <font>
                      <family>Arial</family>
                      <pointsize>12</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>PREV</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="nextButton">
                    <property name="font">
                     <font>
                      <family>Arial</family>
                      <pointsize>12</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>NEXT</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="quitButton">
                    <property name="font">
                     <font>
                      <family>Arial</family>
                      <pointsize>12</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>QUIT</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Log</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="ConsoleWidget" name="pythonConsole" native="true"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1591</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_File"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <italic>true</italic>
    </font>
   </property>
  </widget>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
  <customwidget>
   <class>ConsoleWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph.console</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>894</y>
    </hint>
    <hint type="destinationlabel">
     <x>525</x>
     <y>467</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>frameSelector</sender>
   <signal>valueChanged(int)</signal>
   <receiver>frameNumberSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>855</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>855</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rawDisplayScale</sender>
   <signal>valueChanged(int)</signal>
   <receiver>rawDisplayScaleSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>708</y>
    </hint>
    <hint type="destinationlabel">
     <x>613</x>
     <y>713</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rawDisplayScaleSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>rawDisplayScale</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>613</x>
     <y>713</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>708</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
